src/vivarium_inputs/validation/raw.py

```
@@ -47,30 +64,73 @@ def check_metadata(entity: Union[ModelableEntity, NamedTuple], measure: str):
     metadata_checkers[entity.kind](entity, measure)
```

We should raise an InvalidQueryError here if we get an unknown entity.kind